storage/src/context_action_storage.rs

```diff
@@ -72,6 +73,8 @@ pub struct ContextActionStorage {
     context_by_type_index: ContextActionByTypeIndex,
     kv: Arc<ContextActionStorageKV>,
     generator: Arc<SequenceGenerator>,
+    last_block_hash: Arc<Option<BlockHash>>,
```

Comment:
@adonagy 
please, add here more comments for both attributes, what is stored or not and so on